cfme/fixtures/vm.py

```diff
@@ -113,11 +127,53 @@ def create_vm(setup_provider, request, provider):
     if request.param:
         template_type = request.param
     else:
-        pytest.error('Any appropriate Template was not passed to the fixture.')
+        pytest.error('A VM/Instance template type was not passed to the fixture.')
+
     vm_name = _get_vm_name(request)
     return _create_vm(request, template_type, provider, vm_name)
 
 
+@pytest.fixture(scope="module")
+def create_vms_modscope(setup_provider_modscope, request, provider):
+    """Create one or more VMs/Instances."""
+    template_type = None
+    param = request.param
+
+    if param and isinstance(param, dict):
+        template_type = param.get('template_type', None)
+        num_vms = param.get('num_vms', 1)
+
+    if not template_type:
+        pytest.error('A VM/Instance template type was not passed to the fixture.')
+
+    vms = []
+    for _ in range(num_vms):
+        vm_name = _get_vm_name(request)
+        vm = _create_vm(request, template_type, provider, vm_name)
+        vms.append(vm)
+    return vms
+
+
+@pytest.fixture(scope="function")
+def create_vms(setup_provider, request, provider):
+    """Create one or more VMs/Instances."""
+    template_type = None
+
+    param = request.param
+    if param and isinstance(param, dict):
+        template_type = param.get('template_type', None)
+        num_vms = param.get('num_vms', 1)
+    if not template_type:
+        pytest.error('A VM/Instance template type was not passed to the fixture.')
+
+    vms = []
+    for _ in range(num_vms):
```

Comment:
I'm not seeing a default behavior/setting for this parameter - above the setting of `num_vms` is dependent on the parameter being a dictionary - if its a tuple, this breaks with UnboundLocal.

I think its safe to set a default of 2, since its plural, and otherwise set it to the parametrized value via num_vms

Reply:
Accidentally included this commit in this PR / branch instead of #9873.